tests/functional/bounced_email.js

```diff
@@ -94,4 +101,46 @@ define([
     }
 
   });
+
+  const setUpBouncedSignIn = thenify(function () {
+    const client = new FxaClient(AUTH_SERVER_ROOT, {
```

Comment:
I realize it's done this way above, you can use [helpers.js->getFxaClient](https://github.com/mozilla/fxa-content-server/blob/f9f7f10d4310163ffc256d1536322d470dcfe646/tests/functional/lib/helpers.js#L474:L483) to do this for you.